include/Channel.hpp

```diff
@@ -13,9 +13,16 @@ typedef enum	e_channel_namespace
 	PUBLIC = '#'
 }				t_channel_namespace;
 
+typedef enum	e_channel_flag
+{
+	FLAG_INVITE = 1,
+	FLAG_KEY	= 2
+}				t_channel_flag;
+
 class _base_channel
 {
 	public:
+		_base_channel(void);
```

Comment:
I know I told you to add this constructor, but it is not needed anymore, and it is quite useless as the `Channel`'s fields are const